certification/pyxis/pyxis.go

```diff
@@ -401,5 +379,5 @@ func (p *pyxisClient) newRequest(ctx context.Context, method string, url string,
 
 // checkStatus is used to check for a 2xx status code
 func checkStatus(statusCode int) bool {
-	return statusCode >= 200 && statusCode < 300
+	return statusCode >= http.StatusOK && statusCode < http.StatusMultipleChoices
```

Comment:
This strikes me as a super odd way to check for success. The function signature also doesn't make sense. Sorry, I know this isn't in scope for this PR, it just drew my attention to it. Might be a future quick patch.